core/src/test/java/com/nextdoor/bender/testutils/DummyThrottleOperationHelper.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * Copyright 2018 Nextdoor.com, Inc
+ *
+ */
+
+package com.nextdoor.bender.testutils;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.nextdoor.bender.InternalEvent;
+import com.nextdoor.bender.config.AbstractConfig;
+import com.nextdoor.bender.operation.BaseOperation;
+import com.nextdoor.bender.operation.EventOperation;
+import com.nextdoor.bender.operation.OperationConfig;
+import com.nextdoor.bender.operation.OperationFactory;
+
+public class DummyThrottleOperationHelper {
```

Comment:
Document the various new classes we've created.. they aren't all immediately obvious what they do.